### DISCUSSION TITLE
What does Tiled VAE do and do I need it 16VRAM

### DISCUSSION THREAD

**Discussion by sadume** [pkuliyi2015/multidiffusion-upscaler-for-automatic1111#216] [Q&A] [answered]
What does Tiled VAE do and do I need it 16VRAM?

thx

**Comment by pkuliyi2015** [comment, marked as answer]
It just saves your VRAM. If you don't use it, you can only produce no larger than 2048 * 2048 (even smaller if you use controlnet or other extension that takes away your VRAM). With Tiled VAE on, you can go up to 8192 * 8192.

**Reply by andupotorac** [reply]
But what is the difference between Tiled VAE and Tiled Diffusion? When do we use one, and when do we use the other?

This is the most basic question but it's not explained in the main page here on Github, and neither is easy to find in the discussions. The above answer doesn't compare the two either.

**Reply by pkuliyi2015** [reply]
I'm sorry but this is quite technical. 

Basically stable diffusion can be divided into two stages: A vae encoding/decoding stage that converts pixels into latent images(and convert back); A unet denoising stage that generates latent images from random noises.

The tiled vae tiles the first stage, and the tiled diffusion tiles the second stage. Both will help you save VRAM. 

But there are also differences. The tiling of vae doesn't visibly change its output and input. But the tiling of diffusion will greatly change the way diffusion works. I won't elaborate more here as you can see from many failing cases.

You can always use Tiled vae. As for tiled diffusion you need to get familiar with its effect, otherwise it will break your image.

**Reply by andupotorac** [reply]
Thanks for the answer. My question was mainly to understand which of the settings needs to be used, because I keep getting these results and I followed the same values for the parameters as specified on the main page of this extension.

https://github.com/pkuliyi2015/multidiffusion-upscaler-for-automatic1111/discussions/253
